openslides_backend/action/actions/assignment/create_update_delete.py

```diff
@@ -63,6 +81,8 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
                             )
                         ]
                     )["assignment_candidate"].values()
+                    if (meeting_user_id := candidate["meeting_user_id"])
```

Comment:
Why use the walross notation here ? I would prefer using candidate... twice.

Reply:
Changed that but the if statement will still be over two lines.